internal/proto/buf/prototransform/v1alpha1/lease.proto

```diff
@@ -16,8 +16,6 @@ syntax = "proto3";
 
 package buf.prototransform.v1alpha1;
 
-import "google/protobuf/timestamp.proto";
-
```

Comment:
This was unused. It wasn't picked up before because it looks like `buf lint` and `buf format` aren't part of this repo's CI. Will fix in a subsequent PR.